### ISSUE TITLE
player dont get damage in void

### ISSUE THREAD

**Issue by Driesboy** [svilex/SkyWars-PocketMine#95]
## Issue description

when players fall off into the void in the first 20 seconds which says "you can't pvp", they won't get killed by the void and are stuck there until those 20 seconds pass.

## Versions

- **SW_VERSION**:  0,6

- **POCKETMINE**:  Elywings

- **PHP**: 7.0.14

**Comment by svilex** [comment]
Yes, quickly looked at the code, and seems like you are right, there is a mess [right there](https://github.com/svilex/SkyWars-PocketMine/blob/master/SW/src/svile/sw/SWlistener.php#L450-L459)
